symbols/arm9.yml

```diff
@@ -686,7 +686,23 @@ arm9:
           - 0x2052418
           - 0x2054A60
           - 0x20609E8
```

Comment:
This NA address list was actually manual. Could we fill these in too, so we're matching the EU list? In the current commit there are 14 for NA but 16 for EU.

Reply:
I did find those missing copies (4 for EU and 2 for NA) but I didn't add them because they seem to be unreachable code. Ghidra doesn't disassemble them unless you do it manually. It's like in the heap allocation table, there's some valid code there as well but it doesn't seem to be referenced anywhere

Reply:
It's interesting that EU only has 2 whereas NA has 4...

Well, I guess it doesn't matter too much at any rate.